{{cookiecutter.project_slug}}/backend/app/app/crud/__init__.py

```diff
@@ -1 +1,11 @@
-from . import item, user
+from .user import user
+from .item import item
+
+# For a new basic set of CRUD operations, on a new object, let's say 'SubItem',
+# you could also simply add the following lines:
+
+from .base import CrudBase
+from app.models import subitem as models_subitem
+from app.schemas import subitem as schemas_subitem
+
+subitem = CrudBase(models_subitem.SubItem, schemas_subitem.SubItem)
```

Comment:
Is it a good idea to use module variables (effectively globals/singletons)? What happens if you get multiple requests and one of them is using this object?

Reply:
Those variables are just "gateways" from/into the DB. It will be the DB which will make sure that there is no conflict when multiple requests come in.